Client/Frontend/Home/JumpBackIn/FxHomeJumpBackInViewModel.swift

```diff
@@ -37,8 +37,6 @@ class FirefoxHomeJumpBackInViewModel: FeatureFlagsProtocol {
     private lazy var siteImageHelper = SiteImageHelper(profile: profile)
     private var isPrivate: Bool
 
-    private lazy var homescreen = nimbus.features.homescreen.value()
```

Comment:
I think nimbus params can be deleted in FirefoxHomeJumpBackInViewModel, FxHomeTopSitesViewModel, FirefoxHomeViewModel and HomePageSettingViewController since it was used only for homescreen lazy var